### ISSUE TITLE
Request: User should be informed when they are directly linked to and using an older version of the agent

### ISSUE THREAD

**Issue by jayzalowitz** [nearai/nearai#567]
Currently causing problems for the memecoin generator where earlier versions had minting issues.

**Comment by calebjacob** [comment]
Good idea. I'll get something implemented for this quickly. This older PR might also help if we can get it wrapped up: https://github.com/nearai/nearai/pull/412

**Comment by calebjacob** [comment]
We merged and deployed this PR to production which should resolve this issue: https://github.com/nearai/nearai/pull/412

By default, opening an agent will now use the `latest` version: https://app.near.ai/agents/jayzalowitz.near/memecoin_agent

When anyone starts a thread with `latest` selected, their thread will always run against the latest agent source code. This should solve the core issue of users being stuck on old versions of the agent moving forward. A user would need to be sent a direct version link (eg: ` https://app.near.ai/agents/jayzalowitz.near/memecoin_agent/x.x.x`) or manually select a fixed version from the version dropdown. The UI has been updated as well to make it more clear that you're on a fixed version VS latest.

Let me know if you have any questions/concerns. Happy to revisit further, but will close this for now.
